src/Nethermind/Nethermind.Merge.Plugin/Data/BlobsBundleV1.cs

```diff
@@ -0,0 +1,55 @@
+// SPDX-FileCopyrightText: 2022 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+using System;
+using System.Collections.Generic;
+using Nethermind.Core;
+using Nethermind.Core.Crypto;
+using Newtonsoft.Json;
+
+namespace Nethermind.Merge.Plugin.Data;
+
+/// <summary>
+/// Holds blobs of a block.
+///
+/// See <a href="https://github.com/ethereum/execution-apis/blob/main/src/engine/experimental/blob-extension.md#blobsbundlev1">BlobsBundleV1</a>
+/// </summary>
+public class BlobsBundleV1
+{
+    public BlobsBundleV1()
+    {
+    }
+
+    public BlobsBundleV1(Block block)
+    {
+        List<Memory<byte>> commitments = new();
+        List<Memory<byte>> blobs = new();
+        List<Memory<byte>> proofs = new();
```

Comment:
preallocate to block.Transactions.Length
Maybe use ArrayPoolLists to avoid allocation.